test/units/modules/cloud/amazon/placebo_fixtures.py

```diff
@@ -0,0 +1,192 @@
+from nose.plugins.skip import SkipTest
+import pytest
+import os
+
+try:
+    import boto3
+    import botocore
+    import placebo
```

Comment:
The `placebo` package needs to be added to the unit test requirements:

https://github.com/ansible/ansible/blob/devel/test/runner/requirements/units.txt